scripts/publishNPMPackages.js

```diff
@@ -31,6 +31,8 @@ import { invariant } from '@adeira/js';
       '@adeira/test-utils',
       '@adeira/flow-bin',
       '@adeira/eslint-config',
+      'eslint-plugin-adeira',
+      'eslint-plugin-relay-imports',
```

Comment:
I am not sure if this is a good idea since it was used before we migrated and therefore we might be messing with the versions and unexpectedly release something they don't have under control. How about moving this code under `eslint-plugin-adeira`?

Reply:
Ok, I will see if I can move it under `eslint-plugin-adeira`.